core/src/banking_stage/transaction_scheduler/prio_graph_scheduler.rs

```diff
@@ -264,12 +265,12 @@ impl<Tx: TransactionWithMeta> PrioGraphScheduler<Tx> {
                         {
                             schedulable_threads.remove(thread_id);
                             if schedulable_threads.is_empty() {
-                                break;
+                                break 'outer_schedule_loop;
```

Comment:
if there are no schedulable threads, we're done scheduling - don't need to iterate over our graph again.